pkg/bisect/bisect.go

```diff
@@ -110,7 +111,7 @@ func Run(cfg *Config) ([]*vcs.Commit, *report.Report, error) {
 			env.log("the crash already happened on the oldest tested release")
 		}
 		env.log("crash: %v\n%s", rep.Title, rep.Report)
```

Comment:
We need to log the bad commit before the "crash" line. This will allow to test this logic with syz-bisect utility.